src/arch/ofi/machine.C

```diff
@@ -814,7 +817,18 @@ void LrtsInit(int *argc, char ***argv, int *numNodes, int *myNodeID)
       ///      short hsnOrder[numcxi]={2,1,3,0};
       if(numcxi==4)
 	{
-	  short hsnOrder[4]= {1,3,0,2};
+	  short hsnOrder[8]= {1,1,3,3,0,0,2,2};
```

Comment:
Since myRank%quad <= numcxi (here 4) only first 4 elements of hsnOrder would be used, so change in line 820 may be a typo? Otherwise looks good to me.